mtz-url-rewrite-interceptor.html

```diff
@@ -0,0 +1,143 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="mtz-interceptor-behavior.html">
+
+<!--
+`mtz-url-rewrite-interceptor`
+An implementation of the mtz.InterceptorBehavior that injects a header:value onto a presend event for iron-ajax.
+
```

Comment:
I think you commented on an old commit. Also, we shouldn't mention about rewriting for different environments because that is just 1 use case of it. This interceptor is more generic than that.